cmd/versionUpgrade.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var downloadURL = "https://github.com/karimra/gnmic/raw/master/install.sh"
+
+// upgradeCmd represents the version command
+var upgradeCmd = &cobra.Command{
+	Use:   "upgrade",
+	Short: "upgrade gnmic to latest available version",
+
+	Run: func(cmd *cobra.Command, args []string) {
+		f, err := ioutil.TempFile("", "gnmic")
+		defer os.Remove(f.Name())
+		if err != nil {
+			log.Fatalf("Failed to create temp file %s\n", err)
```

Comment:
need to use logger, not log here. or return an error

Reply:
addressed in 7ec3633